homepage/models.py

```diff
@@ -96,3 +97,17 @@ def __str__(self):
         shortened_review = shortened_review[:MAX_LENGTH_PREVIEW]
 
         return f'Preview: {shortened_review}...'
+
+    def print_details(self):
+        message = (
+            f"Course: {self.course}\n"
+            f"User: {self.user}\n"
+            f"Rate: {self.rate}\n"
+            f"{str(self)}\n"

```

Comment:
unclear what this is - is this the entire review content?

Reply:
It is a part of the review content.
I'll add a comment to clarify